From da79ce1df6f31944203faca41db7b57315fcf549 Mon Sep 17 00:00:00 2001
From: Igal Zeltser <izeltser@habana.ai>
Date: Wed, 14 Sep 2022 14:16:00 +0300
Subject: [PATCH] [SW-101040] build: remove misleading indentation warning

New gcc version spits misleading-indentation for no good reason.
Gcc flag is added to remove this.

Change-Id: Iaf6fe8c769ecb475f1e5f2e4319c70cad14b3ef4
Signed-off-by: Igal Zeltser <izeltser@habana.ai>
---

diff --git a/Makefile b/Makefile
index 0d86baf..6fe5dd8 100644
--- a/Makefile
+++ b/Makefile
@@ -15,7 +15,7 @@
 ARCBOOT_VER := 0x0
 
 LIBDIR = sdk/lib
-ARCBOOT_CFLAGS = -Wall -W -Werror -O3 -g
+ARCBOOT_CFLAGS = -Wall -W -Werror -Wno-misleading-indentation -O3 -g
 ARCBOOT_CFLAGS += -Iinclude
 ARCBOOT_CFLAGS += -Icommon
 ARCBOOT_CFLAGS += -Icommon/rsa
